test/bindings/node/data/Makefile

```diff
@@ -0,0 +1,32 @@
+OSRM_RELEASE:=$(shell node -e "console.log(require('../../../../package.json').osrm_release)")
```

Comment:
Yes, requires changing all coordinates from locations in Bonaco to Berlin - I agree having multiple datasets is ugly.